frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template.tsx

```diff
@@ -159,18 +186,16 @@ const VirtualMachineTemplates: React.FC<React.ComponentProps<typeof Table> &
     </div>
   );
 };
+
 const getCreateProps = ({ namespace }: { namespace: string }) => {
   const items: any = {
-    wizard: 'New with Wizard',
-    yaml: 'New from YAML',
+    wizard: VMWizardActionLabels.WIZARD,
```

Comment:
I guess this doesn't hurt as well. But I mean mostly VMWizardName constants for the keys.

Reply:
haha :-)